cypress/support/pages/user.pageObject.js

```diff
@@ -0,0 +1,16 @@
+import PageObject from '../PageObject';
+
+class UserPageObject extends PageObject {
+  url = '/#/';
```

Comment:
The URL in the 'url' property seems to be a base URL, not a specific User Page URL. Please make sure to set the correct URL for the User Page.